examples/src/main/python/mllib/gradient_boosted_trees.py

```diff
@@ -0,0 +1,82 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""
+Gradient boosted Trees classification and regression using MLlib.
+"""
+
+import sys
+
+from pyspark.context import SparkContext
+from pyspark.mllib.tree import GradientBoostedTrees
+from pyspark.mllib.util import MLUtils
+
+
+def testClassification(trainingData, testData):
+    # Train a GradientBoostedTrees model.
+    #  Empty categoricalFeaturesInfo indicates all features are continuous.
+    model = GradientBoostedTrees.trainClassifier(trainingData,
+                                                 categoricalFeaturesInfo={},
+                                                 numIterations=30,
+                                                 maxDepth=4)
```

Comment:
For the code style, we don't chop down arguments in method calls. For example: https://github.com/apache/spark/blob/master/python/pyspark/mllib/tree.py#L137

So this should be 

~~~
    model = GradientBoostedTrees.trainClassifier(trainingData, categoricalFeaturesInfo={},
             numIterations=30, maxDepth=4)
~~~

or 

~~~
    model = GradientBoostedTrees.trainClassifier(
             trainingData, categoricalFeaturesInfo={}, numIterations=30, maxDepth=4)
~~~